libs/shared/npm/src/lib/find-nx-package-path.ts

```diff
@@ -25,7 +25,7 @@ export async function findNxPackagePath(
 
   const nrwlWorkspaceDepPath = await workspaceDependencyPath(
     workspacePath,
-    '@nrwl/workspace'
+    '@nx/workspace'
```

Comment:
here we might also need another case for those not yet on v16 but also without an `nx` package? not sure..